test/compose/test-compose

```diff
@@ -206,6 +206,9 @@ function start_service() {
     mkdir --mode 0755 $WORKDIR/{root,runroot,cni}
     chcon --reference=/var/lib/containers $WORKDIR/root
 
+    # hide annoying podman compose warnings, some tests want to check compose stderr and this breaks it.
+    export PODMAN_COMPOSE_WARNING_LOGS=false
+
```

Comment:
This is not needed, the issue is not that the env is not set because the old env clearly worked.

you can easily reproduce the test issue locally by running `test/compose/test-compose uptwice`